GraphQLUpload.mjs

```diff
@@ -66,7 +66,7 @@ import Upload from "./Upload.mjs";
  * {@linkcode GraphQLUpload} scalar resolver value promise resolves:
  *
  * ```ts
- * import type { FileUpload } from "graphql-upload/processRequest.mjs";
```

Comment:
The `type` word is important. You can learn about it here:

https://www.typescriptlang.org/docs/handbook/modules.html#importing-types

Reply:
The [link](https://web.archive.org/web/20230216042921/https://www.typescriptlang.org/docs/handbook/modules.html#importing-types) is now broken, this is an updated version:

[https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export)